src/Sulu/Bundle/AdminBundle/Resources/js/containers/List/adapters/abstractTableAdapter.scss

```diff
@@ -1,3 +1,7 @@
+.publish-indicator {
```

Comment:
Should that be exactly the same as in the `ghost-indicator` class? Why not using the selector `.publish-indicator, .ghost-indicator` then?